velox/functions/prestosql/Reduce.cpp

```diff
@@ -169,6 +170,11 @@ class ReduceFunction : public exec::VectorFunction {
           &localResult);
     }
     context.moveOrCopyResult(localResult, rows, result);
+    for (int i = 0; i < rows.end(); i++) {
```

Comment:
Does something like below work?
```
for (const auto& row : rows) {
  if (partialResult->isNull(row) {
      result->setNull(i, true);
   }
}
```

Reply:
I tried this. It doesn't work and will cause wrong result for the not NULL arrays.

Reply:
It doesn't work, seems partialResult will changed during merge function and complete function compute.